pages/dkp/konvoy/2.2/release-notes/index.md

```diff
@@ -54,6 +54,13 @@ The default value for flag `--with-aws-bootstrap-credentials` will be changing f
 
 #### New command to update Azure credentials in running clusters
 
+#### Default update strategy changed to "delete first" for Preprovisioned clusters
+
+A "create first" update strategy first creates a new machine, then deletes the old one. While this strategy works when
+machine inventory can grow on demand, it does not work if there is a fixed number of machines. Most Preprovisioned
+clusters have a fixed number of machines. To enable updates for Preprovisioned clusters, DKP uses the "delete first"
```

Comment:
@dlipovetsky where do we describe how to change things in your pre-provisioned env? We need to tell them how to do this if there are manual steps. If not, then we should say we do it for them.

Reply:
New clusters will have the correct strategy by default.

Existing clusters will be automatically updated to use the correct strategy whenever users run the update commands: `update controlplane` and `update nodepool`.

Reply:
ok, let's say 'for more info on updating existing clusters, see xxx topic.

Reply:
I added:
> New clusters use the "delete first" strategy by default. Existing clusters are switched to the "delete first" strategy
whenever their machines are updated.